packages/lit-element/src/lib/css-tag.ts

```diff
@@ -94,5 +90,12 @@ export const css = (
     (acc, v, idx) => acc + textFromCSSResult(v) + strings[idx + 1],
     strings[0]
   );
-  return new CSSResult(cssText, constructionToken);
+  let result = cssResultCache.get(cssText);
```

Comment:
Strongly keying the cache off of the strings means every `CSSResult` will be retained for the life of the program. We need to be sure the benefits (reusing the constructible stylesheet) is worth it. Since these are generally going to be used in CE ctors (which are also retained for the life of the program), probably ok?

Reply:
This seems worth it given the known browser optimizations when sharing styleSheets.

Reply:
I suspect it's going to be really rare that program code is GC'ed